lib/address.ex

```diff
@@ -39,11 +39,31 @@ defmodule Bitcoinex.Address do
     is_valid_base58_check_address?(address, network.p2sh_version_decimal_prefix)
   end
 
-  def is_valid?(address, network_name, address_type) when address_type in [:p2wpkh, :p2wsh] do
+  def is_valid?(address, network_name, :p2wpkh) do
     case Segwit.decode_address(address) do
-      {:ok, {^network_name, _, _}} ->
-        true
+      {:ok, {^network_name, witness_version, witness_program}} ->
```

Comment:
a cleaner way will be 
```
       {:ok, {^network_name, 0, witness_program}} when length(witness_program) == 20 -> 
          true
```
So we can get ride of one unless if expression. Same for line 61

Reply:
Right! Changed